src/types/formatted/qaSection.ts

```diff
@@ -0,0 +1,12 @@
+import { FormattedParagraph } from '@/data/queries'
+import { PublishedParagraph } from './publishedEntity'
+import { HeadingLevel } from './headingElement'
+
+export type QaSection = PublishedParagraph & {
+  header: string
+  intro: string
+  displayAccordion: boolean
+  questions: FormattedParagraph[]
```

Comment:
This can be one of two paragraph types:
![image](https://github.com/department-of-veterans-affairs/next-build/assets/6863534/a9d8ee66-e5c6-4487-944e-4e3050e91946)

I think we should type this accordingly.

Notably, it's a bit interesting that this can be a React Widget. I only see one example of this in the current content, and it's a "test" with "do not publish". But the IA allows for it, so I think we need to accommodate it. It's important to consider that this will likely change how the template works, though, since it will need to handle the possibility that it's a React Widget as well. I think this can be handled by using the general `<Paragraph ...>` component in the template rather than the more specific `<QaParagraph ...>` component.

Reply:
Agree with this,  will plug in paragraph component instead

Reply:
Just a note that Josh and I discussed and agreed that this might be better remaining typed as `questions: FormattedParagraph[]`. There are currently only two paragraph types allowed, but there could theoretically be more added. Typing it more specifically offers some advantages in type safety, IDE type hinting, etc. but this feels like a good tradeoff in this case.